packages/playground-examples/copy/id/JavaScript/Helping with JavaScript/Errors.ts

```diff
@@ -0,0 +1,44 @@
+//// { order: 3, isJavaScript: true }
+
+// TypeScript secara bawaan tidak memeriksa galat
+// yang ada dalam kode JavaScript. Sebaliknya, perkakas
+// TypeScript terfokus pada menyediakan dukungan yang
+// kaya pada editor.
+
```

Comment:
Kalimat terakhir ini masih susah dipahami

Reply:
Kalimat dalam bahasa aslinya adalah `Instead the tooling is focused on providing rich support for editors.`

Bagaimana bila saya terjemahkan ulang menjadi `TypeScript difokuskan untuk memberikan banyak dukungan pada editor.`?

Reply:
Sudah diterjemahkan ulang melalui _commit_ 997a15703910e0a611537aa9a66fb26fd6fe707a